code/datums/bodypart_overlays/bodypart_overlay.dm

```diff
@@ -0,0 +1,82 @@
+///Bodypart ovarlay datum. These can be added to any limb to give them a proper overlay, that'll even stay if the limb gets removed
+///This is the abstract parent, don't use it!!
+/datum/bodypart_overlay
+	///Sometimes we need multiple layers, for like the back, middle and front of the person (EXTERNAL_FRONT, EXTERNAL_ADJACENT, EXTERNAL_BEHIND)
+	var/layers
+	///List of all possible layers. Used for looping through in drawing
+	var/static/list/all_layers = list(EXTERNAL_FRONT, EXTERNAL_ADJACENT, EXTERNAL_BEHIND, EXTERNAL_FRONT_UNDER_CLOTHES, EXTERNAL_FRONT_OVER) // SKYRAT EDIT - Customization - ORIGINAL: var/static/list/all_layers = list(EXTERNAL_FRONT, EXTERNAL_ADJACENT, EXTERNAL_BEHIND)
```

Comment:
Skyrat comment? What happened to this file?

Reply:
This was in the missed mirror, and I then added some changes myself.